models/src/main/java/com/vimeo/networking2/AlbumEmbed.kt

```diff
@@ -0,0 +1,13 @@
+package com.vimeo.networking2
+
+/**
+ * Embed data for a album.
+ */
+data class AlbumEmbed(
+
+    /**
+     * The responsive HTML code to embed the playlist on a website. This is present only
```

Comment:
It is an album, not a playlist. Also, I think it is not present for the free user tier. Check with API to be 100% sure.

Reply:
It is not present for the free tier. I updated the docs.